### PR TITLE
refactor: MW-143 Home UI redesign

### PR DESCRIPTION
**Pull request by itsPronay**
## Issue Fix
Fixes #{Issue Number}
Jira Task: [MW-143](https://mifosforge.jira.com/browse/MW-143)

## Screenshots
New UI | Figma |
---------------|-------------|
image |  image

##

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

[MW-143]: https://mifosforge.jira.com/browse/MW-143?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: feature/home/src/main/kotlin/org/mifospay/feature/home/HomeScreen.kt [hunk 1]
```diff
@@ -110,170 +115,294 @@ private fun HomeScreen(
...
+            Column(
+                modifier = Modifier.fillMaxSize(),
+            ) {
+                Row(
+                    modifier = Modifier.fillMaxWidth(),
+                    horizontalArrangement = Arrangement.SpaceBetween,
+                ) {
+                    Column(modifier = Modifier.padding(20.dp)) {
+                        Text(
+                            text = "Wallet",
```
Comment (niyajali) [inline feature/home/src/main/kotlin/org/mifospay/feature/home/HomeScreen.kt]: Show client name instead..

File: feature/home/src/main/kotlin/org/mifospay/feature/home/HomeScreen.kt [hunk 2]
```diff
@@ -110,170 +115,294 @@ private fun HomeScreen(
...
-            TransactionItemScreen(transaction = transaction)
-        }
-        item {
-            Spacer(modifier = Modifier.height(24.dp))
-        }
     }
 }
 
 @Composable
-private fun MifosWalletCardScreen(
```
Comment (niyajali) [inline feature/home/src/main/kotlin/org/mifospay/feature/home/HomeScreen.kt:147]: This Composable could be private or internal

**Review by niyajali — CHANGES_REQUESTED** [review]

**Comment by niyajali** [comment]
@itsPronay update the screenshot based on latest changes

**Comment by itsPronay** [comment]
> @itsPronay update the screenshot based on latest changes

done

File: feature/home/src/main/kotlin/org/mifospay/feature/home/HomeScreen.kt [hunk 3]
```diff
@@ -110,170 +115,297 @@ private fun HomeScreen(
...
+            Column(
+                modifier = Modifier.fillMaxSize(),
+            ) {
+                Row(
+                    modifier = Modifier.fillMaxWidth(),
+                    horizontalArrangement = Arrangement.SpaceBetween,
+                ) {
+                    Column(modifier = Modifier.padding(20.dp)) {
+                        Text(
+                            text = "Client Name",
```
Comment (niyajali) [inline feature/home/src/main/kotlin/org/mifospay/feature/home/HomeScreen.kt:186]: Show the Client Name @itsPronay

**Review by niyajali — CHANGES_REQUESTED** [review]

File: feature/home/src/main/kotlin/org/mifospay/feature/home/HomeScreen.kt (hunk 3 above)
Comment (niyajali) [reply feature/home/src/main/kotlin/org/mifospay/feature/home/HomeScreen.kt:186]: Don't hardcode the text, show client name instead

Comment (itsPronay) [reply feature/home/src/main/kotlin/org/mifospay/feature/home/HomeScreen.kt:186]: image
we are already showing here.

Comment (itsPronay) [reply feature/home/src/main/kotlin/org/mifospay/feature/home/HomeScreen.kt:186]: in the demo API, `account?.name = WALLET`

Comment (niyajali) [reply feature/home/src/main/kotlin/org/mifospay/feature/home/HomeScreen.kt:186]: Oh! Okay

**Pull request merged**